source/agora/network/NetworkClient.d

```diff
@@ -186,6 +187,9 @@ class NetworkClient
     {
         try
         {
+            if (!this.api.getNodeInfo().is_validator)
```

Comment:
This change has several problems:
- sendEnvelope() was previously an asynchronous task. You have now turned it into a synchronous (blocking) task. Naturally this will break all the unittest code that uses timeouts.
- You should query the node whether it is a validator **once** after you connect to it. What is the purpose of asking it if it's still a validator every time we gossip an envelope? The node won't magically turn into a FullValidator while the connection is still active.